editions/tw5.com/tiddlers/definitions/TiddlyFox.tid

```diff
@@ -1,10 +1,10 @@
 created: 20130825161100000
-modified: 20200104111952539
+modified: 20230803051056946
 tags: Definitions
 title: TiddlyFox
 type: text/vnd.tiddlywiki
 
```

Comment:
IMO this tiddler should start with

```
<<.deprecated-since "FireFox 57" "Saving">>
```

Reply:
The links at line 13ff are broken. @Jermolene -- I think they should be removed

Reply:
Depreciation added (here and other TiddlyFox tid).